### DISCUSSION TITLE
How to migrate all the bitbucket projects to github enterprise?

### DISCUSSION THREAD

**Discussion by ssimarpreet285** [community/community#40612] [Enterprise]
### Select Topic Area

Question

### Body

Hello,

We have similar to 150 projects in bitbucket which we want to migrate over Github EMU. Can someone please layout a step-by-step process to achieve this without losing any data.

Thanks

**Comment by ossanna16** [comment]
Hi there @ssimarpreet285 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
